Documentation clarifications (#5889)
yarnpkg/berry: packages/docusaurus/docs/getting-started/basics/install.md

The preferred way to manage Yarn is by-project and through [Corepack](https://nodejs.org/dist/latest/docs/api/corepack.html), a tool shipped by default with Node.js. Modern releases of Yarn aren't meant to be installed globally, or from npm.
:::

> 1. Start by enabling Corepack, if it isn't already; running this command will cause the `yarn` binary to be added in your `$PATH`:

Reviewer: ```suggestion
1. Start by enabling Corepack, if it isn't already; running this command will add the `yarn` binary to your `$PATH`:
```
